diff --git a/app/.unjs/app.config.ts b/app/.unjs/app.config.ts
index 423d9ad9818bd860edd5f6680b19a8710b5a9ad5..485bb96d5e417554ef2f56e4d8bdd1350e61765a 100644
--- a/app/.unjs/app.config.ts
+++ b/app/.unjs/app.config.ts
@@ -3,8 +3,5 @@ export default defineAppConfig({
     socialBackground: 'https://github.com/unjs/docs/blob/main/assets/ellipse.png?raw=true',
     logo: '/icon.svg',
     github: undefined,
-    socials: {
-      x: 'unjsio',
-    },
   },
 })
diff --git a/app/tailwind.config.ts b/app/tailwind.config.ts
index e9c21b6ed0e7c9636014e9d53d891f6a88700cb9..cc13a3c1221b9fc2d56dd7bac0b90fb81ffeeecf 100644
--- a/app/tailwind.config.ts
+++ b/app/tailwind.config.ts
@@ -25,22 +25,18 @@ export default <Partial<Config>>{
   },
   theme: {
     extend: {
-      fontFamily: {
-        sans: ['Nunito', 'sans-serif', ...defaultTheme.fontFamily.sans],
-      },
       colors: {
-        yellow: {
-          50: '#FEFDF7',
-          100: '#FDFCEF',
-          200: '#FAF6D6',
-          300: '#F7F1BD',
-          400: '#F2E78C',
-          500: '#ECDC5A',
-          600: '#D4C651',
-          700: '#8E8436',
-          800: '#6A6329',
-          900: '#47421B',
-          950: '#2F2C12',
+        pink: {
+          50: '#ff46c5',
+          100: '#ff3cbb',
+          200: '#ff32b1',
+          300: '#ff28a7',
+          400: '#ff1e9d',
+          500: '#ff1493',
+          600: '#f50a89',
+          700: '#eb007f',
+          800: '#e10075',
+          900: '#d7006b',
         },
       },
     },
diff --git a/cli/main.mjs b/cli/main.mjs
index d3f5d078a28bf4bf6291ed340434593367413e77..e57cb954b47ea5d44f066a17021a5cb05a54f64b 100755
--- a/cli/main.mjs
+++ b/cli/main.mjs
@@ -10,6 +10,7 @@ const cli = createCLI({
       github: 'unjs',
       themeColor: 'amber',
     },
+    extends: [process.cwd()],
   },
 })
 
