diff --git a/app/app.config.ts b/app/app.config.ts
index dda72ad13497016c2379db761b35cbcef81949a2..11fa766166a1086d976c4a4fb8b4f9b12c3bda3d 100644
--- a/app/app.config.ts
+++ b/app/app.config.ts
@@ -3,9 +3,6 @@ export default defineAppConfig({
     socialBackground: 'https://github.com/unjs/docs/blob/main/assets/ellipse.png?raw=true',
     logo: '/icon.svg',
     github: undefined,
-    socials: {
-      x: 'unjsio',
-    },
   },
   ui: {
     icons: {
diff --git a/app/tailwind.config.ts b/app/tailwind.config.ts
index b205c02af0f3e161e7fdb30dedc2e34a20ec3b98..b0e6a337560bd8801d3db3022a7bce02d2a78e58 100644
--- a/app/tailwind.config.ts
+++ b/app/tailwind.config.ts
@@ -24,18 +24,17 @@ export default <Partial<Config>>{
         custom: ['Inter'],
       },
       colors: {
-        yellow: {
-          50: '#FEFDF7',
-          100: '#FDFCEF',
-          200: '#FAF6D6',
-          300: '#F7F1BD',
-          400: '#F2E78C',
-          500: '#ECDC5A',
-          600: '#D4C651',
-          700: '#8E8436',
-          800: '#6A6329',
-          900: '#47421B',
-          950: '#2F2C12',
+        pink: {
+          50: '#ff46c5',
+          100: '#ff3cbb',
+          200: '#ff32b1',
+          300: '#ff28a7',
+          400: '#ff1e9d',
+          500: '#ff1493',
+          600: '#f50a89',
+          700: '#eb007f',
+          800: '#e10075',
+          900: '#d7006b',
         },
       },
     },
diff --git a/cli/main.mjs b/cli/main.mjs
index d3f5d078a28bf4bf6291ed340434593367413e77..f5f2b6c5dfbb778c1360284aaf86425c10169ae2 100755
--- a/cli/main.mjs
+++ b/cli/main.mjs
@@ -11,6 +11,7 @@ const cli = createCLI({
       themeColor: 'amber',
     },
+    extends: [process.cwd()],
   },
 })

 cli.runMain()
